git_pile/git_pile.py

```diff
@@ -84,6 +84,20 @@ def assert_required_tools():
 
 
 class Config:
+    __attr_doc_dir = "(path): Directory with PILE_BRANCH checkout (path)"
+    __attr_doc_linear_branch = (
+        "(string): Name of the linear branch to be generated by genlinear-branch. See `genlinear-branch --branch`"
+    )
+    __attr_doc_result_branch = "(string): Name of the branch generated by genbranch. Also see `genbranch --branch`"
+    __attr_doc_pile_branch = '(string): Local name of the PILE_BRANCH - usually "pile"'
+    __attr_doc_format_add_header = "(string): Additional email header to be added on each patch. See `--add-header`"
+    __attr_doc_format_output_directory = "(path): Default output directory. Also see `--output`"
+    __attr_doc_format_compose = "(bool): Invoke a text editor. See `--compose`"
+    __attr_doc_format_signoff = "(bool): Add Signed-off-by trailer. See `--signoff`"
+    __attr_doc_genbranch_committer_date_is_author_date = "(bool): Set committer date as the author date for generated patches. See --committer-date-is-author-date in GIT-COMMIT(1)"
+    __attr_doc_genbranch_user_name = "(string): Name to use as committer when generating the commits"
+    __attr_doc_genbranch_user_email = "(string): E-mail to use as committer when generating the commits"
```

Comment:
In the future, maybe `Config` could be turned into a dataclass and then we could use the `metadata` keyword of [`field()`](https://docs.python.org/3/library/dataclasses.html#dataclasses.field) to store documentation.

Reply:
I will keep it out for now. Using dataclass involves raising the minimum python version (currently 3.6)